Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -70,38 +110,84 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
           zOrigin > m_config.collisionRegionMax) {
         continue;
       }
-      state.compatBottomSP.push_back(bottomSP);
+      // cut on A and B impact parameter coefficients
```

Comment:
Could you spell that out in a little more detail? it's not clear to me from this calculation what A and B impact parameter coefficient are. I freely admit that's because I never fully understood the linear circle transformation :)

Reply:
Hi, I added some comments on the code. This is a cut on the maximum allowed curvature considering the bottom or top SP and the interaction point. We are considering a frame where the central space point SPm is in the origin of the frame and the x axis points away from the interaction point `(xIP ~= -rM, yIP ~= impactParam)`. 

Then, we apply a circle transformation where the circle containing the SP in `x-y` space takes the form of a linear function `v = A * u + B` in the `u-v` space with `u = x / (x² + y²), v = y / (x² + y²)`.
`A` can be obtained by the slope of the linear equation, and `B` by inserting `A` into the linear equation.

Using the circle equation we can also show that `A` and `B` are related to the radius of the circle by `B^2 / (1 + A^2) = 1 / (radius^2)` and this is used apply the cut on the curvature (`minHelixDiameter2`) without the extra conversions and divisions.